### ISSUE TITLE
Inventory supplier stock count

### ISSUE THREAD

**Issue by jphansendk** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#1640]
It would be really nice if we could have a "Remote stock count" on inventory items (like spare parts). That way we can keep an eye on low stock from our suppliers. Of cause, this is only possible if we get the remote stock count from them.. But in our case, we do get it.

**Comment by Uzair-RepairDesk** [comment]
Hey, @jphansendk  hope you are doing well, can you please explain for better understanding.

Regards,
Humza

**Comment by jphansendk** [comment]
Sure.. We receive updates from our supplier 4-6 times a day about their stock of spare parts. We would like to be able to add that to the inventory on RepairDesk. This info is useful to us if we need to order parts and they are low on stock. We can then decide to use another supplier or pick an alternative part. This feature would be great to have - and maybe be able to enable/disable for each supplier. This data will only be updated manually/via external info and will not affect any other info/stock on RepairDesk. Does this make sense?

> Hey, @jphansendk hope you are doing well, can you please explain for better understanding.
> 
> Regards,
> Humza

**Comment by Uzair-RepairDesk** [comment]
Hey @jphansendk  hope you are doing well, you basically want to have an option where you can add multiple suppliers and their respective rates so that at the time of ordering prats you can decide that from which supplier to order?
this is what i get it, correct me if i am wrong.

Regards,
Humza

**Comment by jphansendk** [comment]
No.. just an extra stock counter that is just meant for our suppliers stock.. It does not need to be adjusted by anything other than manual update.. Actually really simple
